static/recommendedBrowsers.html

```diff
@@ -0,0 +1,25 @@
+<html>
+<head>
+    <link rel="stylesheet" href="../css/all.css"/>
+    <!--#include virtual="/title.html" -->
+</head>
+<body>
+    <div class = 'unsupported-desktop-browser'>
+        <h2 class = 'unsupported-desktop-browser__title'>
+            It looks like you're using a browser we don't fully support.
+        </h2>
+        <p class ='unsupported-desktop-browser__description'>
+            We recommend to try with the latest version of&nbsp;
+            <a
+                className = 'unsupported-desktop-browser__link'
+                href = 'http://google.com/chrome' >Chrome</a>,&nbsp;
+            <a
+                class = 'unsupported-desktop-browser__link'
+                href = 'http://www.getfirefox.com/'>Firefox</a>&nbsp;or&nbsp;
+            <a
+                class = 'unsupported-desktop-browser__link'
+                href = 'http://www.chromium.org/'>Chromium</a>
```

Comment:
I want to verify that you want chromium on the list as well.

Reply:
Yes. This is intentional. Actually RTCBrowserType is treating Chromium as Chrome.